browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -439,6 +439,11 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
       {"requestOTRDefault", IDS_SETTINGS_REQUEST_OTR_DEFAULT},
       {"requestOTRAlways", IDS_SETTINGS_REQUEST_OTR_ALWAYS},
       {"requestOTRNever", IDS_SETTINGS_REQUEST_OTR_NEVER},
+#endif
+#if BUILDFLAG(IS_WIN)
+      {"blockWindowsRecallLabel", IDS_SETTINGS_BLOCK_WINDOWS_RECALL_LABEL},
```

Comment:
`block` -> `disabled` for consistency in the code. We can use "block" in the text for the user if we want, but it's easier to search the code if we have consistent naming there. Alternatively you could change it to block everywhere, but we're mostly using disabled right now